### PR TITLE
support runtime-rs ci

### PR DESCRIPTION
**Pull request by yqleng1987**
support runtime-rs ci including compile and test

### CODE REVIEW

File: .ci/run.sh [hunk 1]
```diff
@@ -136,6 +136,18 @@ case "${CI_JOB}" in
...
 		sudo -E PATH="$PATH" bash -c "make filesystem"
 		;;
+	"DRAGONBALL")
+		# TODO: currently runtime-rs binary doesn't support nydus, so disable nydus testing.
+		# Please refer to the issue: https://github.com/kata-containers/kata-containers/issues/4690
+		# echo "INFO: Running nydus test"
+		# sudo -E PATH="$PATH" CRI_RUNTIME="containerd" bash -c "make nydus"
+		echo "INFO: Running stability test"
+		sudo -E PATH="$PATH" CRI_RUNTIME="containerd" bash -c "make dragonball-stability"
+		echo "INFO: Containerd checks"
```
Comment (liubin) [inline .ci/run.sh:146]: Nit: `checks` -> `test`

File: Makefile [hunk 2]
```diff
@@ -73,6 +73,11 @@ stability:
 	ITERATIONS=2 MAX_CONTAINERS=20 ./soak_parallel_rm.sh
 	cd stability && ./hypervisor_stability_kill_test.sh
 
+# If hypervisor is dragonball, the default path to keep pod info is /run/kata. Meanwhile, there is 
+# no independent hypervisor process for dragonball, so disale hypervisor_stability_kill_test.sh
```
Comment (liubin) [inline Makefile:77]: disale -> disable?

**Comment by yqleng1987** [comment]
please refer to https://github.com/kata-containers/tests/pull/4986

**Pull request closed without merging**